dbms/src/DataStreams/HashJoinProbeBlockInputStream.cpp

```diff
@@ -173,7 +149,7 @@ Block HashJoinProbeBlockInputStream::getOutputBlock()
                 {
                     if (join->isEnableSpill())
                     {
-                        block = !join->isRestoreJoin() ? children.back()->read() : restore_probe_stream->read();
+                        block = current_probe_stream->read();
```

Comment:
if join->isEnableSpill() is true but spilling to disk did not happen，current_probe_stream may be null ？